### ISSUE TITLE
some additional shortcuts for the wiki

### ISSUE THREAD

**Issue by allefeld** [mozilla/pdf.js#11475]
The list of keyboard shortcuts in the [wiki](https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#faq-shortcuts) is incomplete, by its own admission. ;)

Here are some additional shortcuts that should be mentioned:

`ctrl+f ` Find in Document
`ctrl+g`  Find next
`shift+ctrl+g` Find previous
`ctrl-s`  Save (Download)
`ctrl-p`  Print
`ctrl-o`  Open File

**Comment by Snuffleupagus** [comment]
> `ctrl+f ` Find in Document
> `ctrl+g` Find next
> `shift+ctrl+g` Find previous
> `ctrl-s` Save (Download)
> `ctrl-p` Print
> `ctrl-o` Open File

Given that those are all *standard* browser shortcuts, which have *exactly* the same meaning in PDF.js as in the browser, it seems questionable if listing those are actually necessary (or even a good idea generally, since none of those shortcuts are handled *directly* in the PDF Viewer in Firefox).

**Comment by allefeld** [comment]
@Snuffleupagus, you really don't like users, do you? When a user asks for the documentation to be amended, doing so is generally a good idea, you know, for the user's benefit? Unless you are not actually developing for users. In that case, maybe disable submitting issues for users?

How *not* documenting something, through a few more lines in the Wiki, can actually be a bad idea is beyond me.

These shortcuts _are_ handled directly in `viewer.js`
`ctrl+f` line 2380
`ctrl+g` & `shift+ctrl+g` line 2388
`ctrl-s` line 2462
Moreover, `ctrl-s` does something substantially different than a standard browser save, which would save the HTML, not the PDF.

`ctrl-p` apparently cannot be handled directly, so you are right. You do override the print handler though, on line 15558, and again, not the HTML but the PDF is handled.

And I was wrong about `ctrl-o`.

**Comment by timvandermeij** [comment]
> you really don't like users, do you?

I don't know what this is about because I don't see any other comments here than this one, but I really want to emphasize that practically all people here are working on PDF.js voluntarily in their spare time, so please keep in mind that everyone is trying their best and refrain from making comments personal.

I have updated https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#faq-shortcuts with the mentioned shortcuts. Thank you for providing input on this subject.

**Comment by allefeld** [comment]
Thank you!

> I don't know what this is about

It was in response to a comment which I found quite personal myself (since deleted).
I should have ignored that. I have now deleted mine, too.

> voluntarily in their spare time

Yes, I realize that and I appreciate it, actually a lot. Without your work, my [Atom package](https://atom.io/packages/pdfjs-viewer) wouldn't be possible, at all. I plan to use it for years to come, and maybe others will, too.

But I found out about those shortcuts voluntarily in my spare time, too. I thought such an (admittedly tiny) contribution was invited by the remark in the wiki that the list is incomplete. But then I'm given the impression – by a specific person whom I won't mention again – that this (admittedly tiny) contribution is utterly pointless, if not outright harmful. I don't understand that attitude.

Apologies for the personalization, and thanks again!
